cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/common/CubeSinkConfig.java

```diff
@@ -42,9 +40,7 @@
     "provided as a JSON Map. For example, if aggregations are desired on fields 'abc' and 'xyz', " +
     "the property should have the value: " +
     "\"{\"dataset.cube.aggregation.agg1.dimensions\":\"abc\", \"dataset.cube.aggregation.agg2.dimensions\":\"xyz\"}. " +
-    "See Cube dataset configuration " +
-    "details for more information : http://docs.cask.co/cdap/current/en/developers-manual/building-blocks/datasets/" +
-    "cube.html#cube-configuration")
+    "See Cube dataset configuration details for more information")
```

Comment:
Perhaps say "See Cube dataset configuration details available at http://docs.cask.co for more information." ?